src/tikzplotly/_save.py

```diff
@@ -92,6 +106,79 @@ def get_tikz_code(
+
+            if trace.name and trace['showlegend'] != False:
+                data_str.append(tex_add_legendentry(sanitize_tex_text(trace.name)))
+
+        elif trace.type == "scatterpolar" or trace.type == "scatterpolargl":
+            data_name_macro, theta_col_name, r_col_name = get_polar_coord(trace, axis, data_container)
+            theta_col_name = "x"
+
+            polar_code = draw_scatterpolar(data_name_macro, theta_col_name, r_col_name, trace, axis, colors_set)
+            data_str.append(polar_code)
+
+            if trace.name and trace['showlegend'] != False:
+                data_str.append(tex_add_legendentry(sanitize_tex_text(trace.name)))
+
+        elif trace.type == "scatter3d":
+            # Handle the case where x, y, or z is empty
+            if trace.x is None or trace.y is None or trace.z is None:
+                warn("Adding empty 3D trace.")
+                data_str.append("\\addplot3 coordinates {};\n")
+                continue
+
+            # View
+            if hasattr(figure_layout.scene, "camera") and hasattr(figure_layout.scene.camera, "eye"):
+                eye = figure_layout.scene.camera.eye
+                if eye is not None and eye.x is not None and eye.y is not None and eye.z is not None:
+                    norm = np.sqrt(eye.x**2 + eye.y**2 + eye.z**2)
+                    azimuth = np.degrees(np.arctan2(eye.y, eye.x))
+                    elevation = np.degrees(np.arcsin(eye.z / norm))
+                    axis.add_option("view", f"{{{azimuth:.1f}}}{{{elevation:.1f}}}")
+
+            # Labels
+            if hasattr(figure_layout.scene.xaxis, "title") and getattr(figure_layout.scene.xaxis.title, "text", None):
+                axis.add_option("xlabel", f"{{{sanitize_tex_text(figure_layout.scene.xaxis.title.text)}}}")
+            if hasattr(figure_layout.scene.yaxis, "title") and getattr(figure_layout.scene.yaxis.title, "text", None):
+                axis.add_option("ylabel", f"{{{sanitize_tex_text(figure_layout.scene.yaxis.title.text)}}}")
+            if hasattr(figure_layout.scene.zaxis, "title") and getattr(figure_layout.scene.zaxis.title, "text", None):
+                axis.add_option("zlabel", f"{{{sanitize_tex_text(figure_layout.scene.zaxis.title.text)}}}")
+
+            # Grid
+            if hasattr(figure_layout.scene.xaxis, "showgrid"):
+                if figure_layout.scene.xaxis.showgrid is False:
+                    axis.add_option("xmajorgrids", "false")
+            if hasattr(figure_layout.scene.yaxis, "showgrid"):
+                if figure_layout.scene.yaxis.showgrid is False:
+                    axis.add_option("ymajorgrids", "false")
+            if hasattr(figure_layout.scene.zaxis, "showgrid"):
+                if figure_layout.scene.zaxis.showgrid is False:
+                    axis.add_option("zmajorgrids", "false")
+
+            # Title
+            if hasattr(figure_layout.scene, "title") and getattr(figure_layout.scene.title, "text", None):
```

Comment:
The attribute `fig.layout.scene.title` does not seem to exist. Do you have an example of code where this is used?